src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/edit-relationship.component.ts/edit-relationships.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { DotMessageService } from '@services/dot-messages-service';
+import { PaginatorService } from '@services/paginator';
+import { EditContentTypeCacheService } from '@portlets/content-types/services/edit-content-type-cache.service';
+import { Relationship } from '@portlets/content-types/fields/shared/relationship.model';
+import { of as observableOf, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { RelationshipCardinality } from '@portlets/content-types/fields/shared/relationship-cardinality.model';
+import { RelationshipService } from '@portlets/content-types/fields/service/relationship.service';
+
+interface CardinalitySorted {
+    [id: number]: RelationshipCardinality;
+}
+@Component({
+    providers: [PaginatorService],
+    selector: 'dot-edit-relationships',
+    templateUrl: './edit-relationships.component.html'
+})
+export class EditRelationshipsComponent implements OnInit {
+
+    @Output()
+    change: EventEmitter<any> = new EventEmitter();
+
+    relationshipsCurrentPage: Observable<{label: string, relationship: Relationship}[]>;
```

Comment:
You are already in a `dot-edit-relationships` you don't need to add the prefix "relationships" to properties.